eng/pipelines/release-cli.yml

```diff
@@ -585,6 +585,7 @@ stages:
 
         variables:
           BaseUrl: http://127.0.0.1:8080
+          AZD_SKIP_UPDATE_CHECK: true
```

Comment:
I'd set this variable at the `stage` level. This only sets it in the install script verification jobs, not the MSI verification jobs.

https://learn.microsoft.com/en-us/azure/devops/pipelines/process/stages?view=azure-devops&tabs=yaml#specify-stages